pkgs/development/python-modules/copykitten/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  fetchFromGitHub,
+  buildPythonPackage,
+  rustPlatform,
+  stdenv,
+  lib,
+  pytestCheckHook,
+  pillow,
+}:
+
+buildPythonPackage rec {
+  pname = "copykitten";
+  version = "1.2.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Klavionik";
+    repo = pname;
```

Comment:
```suggestion
    repo = "copykitten";
```
- #277994

Reply:
I also switched to using a literal string for `pythonImportsCheck`